docs/dev/fundamentals/rollups.md

```diff
@@ -0,0 +1,41 @@
+# Rollups
+
+## Introduction
+
+To better understand rollups we need to dive briefly into Ethereum and Layer 2 solutions.
+
+The design of Ethereum is such that when the demand from users to transact increases, a limit is placed on the network capacity, thereby causing congestion, which results in slow transactions and increased gas prices.
+While this has remained so for a while, an improved solution is needed, one that can maintain scalability.
```

Comment:
The same goes here. Scalability is a property of a system, of its design. It can not be "lost" or "maintained". Solution X can be more scalable than solution Y. People may also say that solution X is not scalable (it scales poorly).